### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by AkkFiros**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -90,8 +104,8 @@ the help window.
 :exclamation: **Warning:**<br>
 
 Upon launching the application, some files responsible for the storage of your data will be created in the same folder 
-as `nutrigoals.jar`. If you are a new user, you are advised not to edit these files. If your changes to the data file 
-makes its format invalid, NutriGoals will discard all your data and start with an empty data file.
+as `nutrigoals.jar`. If you are a new user, you are advised not to edit these files. If the changes you made to the data
+file invalidates its format, NutriGoals will discard all your data and start with an empty data file.
```
Comment (nicolelim02) [inline docs/UserGuide.md]:
```suggestion
as `nutrigoals.jar`. If you are a new user, you are advised not to edit these files. If the changes you made to the data file invalidates its format, NutriGoals will discard all your data and start with an empty data file.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -382,25 +415,24 @@ Format: `exit`
...
+**A**: To download Java `11`, visit this [website](https://www.oracle.com/java/technologies/downloads/#java11) and download the 
+appropriate file for your computer's system (such as macOS and Windows).
 
-**Q**: How do I save my NutriGoals data?<br>
-**A**: NutriGoals data are saved automatically after any command that changes the data. There is no need to save manually.
+**Q**: How do I save my data in NutriGoals?<br>
+**A**: NutriGoals saves data automatically after every command that changes the data. There is no need to save manually.
 
 **Q**: Where is my NutriGoals data stored?<br>
 **A**: When you use the application, a folder named `Data` will be created in the same folder as `nutrigoals.jar`. Your
```
Comment (nicolelim02) [inline docs/UserGuide.md]:
```suggestion
**A**: When you use the application, a folder named `data` will be created in the same folder as `nutrigoals.jar`. Your 
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -62,18 +65,29 @@ This section provides a summary of the parameters used for the commands in this
...
+<div markdown="block" class="alert alert-info">
+
+:information_source: **How to check if you have Java 11 installed:**<br>
+
+* In your search bar, search for 'command prompt' and run it.
+* In the command prompt, type 'java -version' (without the quotation marks).
+* If installed, the version number will be displayed.
+* If not installed, or if the version number is wrong, you can download the correct version from [here](https://www.oracle.com/sg/java/technologies/downloads/#java11-windows).
+
+</div>
```
Comment (ruiqi7) [inline docs/UserGuide.md]: Would it be better to place this in the FAQ? This also seems quite similar to the first question in the FAQ.

Was just thinking that there could be quite a huge gap between step 1 and 2 if we insert this note block here. Not a major issue, just a suggestion to consider! :smiley:

File: docs/UserGuide.md [hunk 4]
```diff
@@ -90,8 +104,8 @@ the help window.
 :exclamation: **Warning:**<br>
 
 Upon launching the application, some files responsible for the storage of your data will be created in the same folder
```
Comment (xav168) [inline docs/UserGuide.md]:
```suggestion
Upon launching the application, some files responsible for the storage of your data will be created in a folder called 'data' located in the same folder 
```

File: docs/UserGuide.md (hunk 3 above)
Comment (AkkFiros) [reply docs/UserGuide.md]: Yes, agreed. Have made the respective changed by adding a link to the FAQ section instead of the note block.

**Review by ruiqi7 — APPROVED** [review]
lgtm!

**Pull request merged**